examples/solvers/chambolle_pock_tomography.py

```diff
@@ -26,8 +26,8 @@
 # Make a parallel beam geometry with flat detector
 # Angles: uniformly spaced, n = 360, min = 0, max = pi
 angle_partition = odl.uniform_partition(0, np.pi, 360)
-# Detector: uniformly sampled, n = 558, min = -30, max = 30
-detector_partition = odl.uniform_partition(-30, 30, 558)
+# Detector: uniformly sampled, n = 512, min = -30, max = 30
```

Comment:
Yes, and in particular anything FBP-like will run significantly faster with power-of-2 sizes (at least if no padding is used).